src/main/resources/public/js/activity.service.js

```diff
@@ -84,9 +84,10 @@
       });
       let i = 0;
       while (i < matches.length) {
-        if (activity.timePerKmInSeconds <= matches[i += 1].timePerKmInSeconds) {
+        if (activity.timePerKmInSeconds <= matches[i].timePerKmInSeconds) {
           break;
         }
+        i += 1;
```

Comment:
@sepe81 A while loop cannot be defined in configuration like a for loop, so the rule no-plus-plus bring an error and thats why I added line 90.